CSSS/Kernel.cs

```diff
@@ -157,6 +157,43 @@ public bool PerformTasks()
                 // so return 'true' from this function to let the
                 // calling main run loop know it can exit
                 shouldExit = true;
+
+                // If we should shutdown
+                if (config.CSSSProgramMode.HasFlag(Config.CSSSModes.Shutdown))
+                {
+                    // Run the shutdown command
+                    var process = new System.Diagnostics.Process();
+                    process.StartInfo.FileName = "shutdown";
+
+                    Console.WriteLine("Shutting down computer in 1 minute for image capture");
+
+                    logger.Info("Attempting to shut down the computer in 1 minute");
+                    switch (config.operatingSystemType)
+                    {
+                        case Config.OperatingSystemType.WinNT:
+                            logger.Info("Operating System detected as WinNT - shutting down computer in 5 seconds");

```

Comment:
Could this logger statement be removed? It has an incorrect value and repeats the information from above.